lookup-field-flow/pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.vaadin.componentfactory</groupId>
     <artifactId>lookup-field-flow</artifactId>
-    <version>23.1.3</version>
+    <version>23.0.2</version>
```

Comment:
I think it should be 23.2.0 and not 23.0.2, am I right?

Reply:
Yes, you're right.

Reply:
Done